packages-common/pluggable-widgets-generator/.eslintrc.js

```diff
@@ -0,0 +1,248 @@
+module.exports = {
```

Comment:
We need to ignore the templates folder for linting and our environment is `node` and not `browser`, so we cannot get rid of having an extra file in generator package. Besides that, PIW-tools eslint rules include quite a lot specific rules for widgets (e.g. jsx, react & jest rules). These rules aren't relevant for the generator, since this isn't a widget. Semantically seen, it would be strange to apply widget linting to generator.

Reply:
We can extend + override. Other option is not to apply it, since to me it seems that the overhead of having this huge eslintrc might very well be not worth the benefit - linting of three js files of which 2 are basically jsons.

Reply:
Linting already proved to be quite useful during refactoring, so I would like to keep it. It reuses PIW-tools config now.